editor/src/components/editor/insert-callbacks.ts

```diff
@@ -134,32 +173,266 @@ function useEnterDrawToInsertForElement(elementFactory: (newUID: string) => JSXE
   )
 }
 
-export function useToInsert(): (convertTo: InsertMenuItem | null) => void {
+const insertAsAbsoluteStrategy = (
```

Comment:
if we keep the strategie(s), please move them to dedicated file(s)

Reply:
[move strategies to separate files](https://github.com/concrete-utopia/utopia/pull/4344/commits/ced8a9282bbb430f89b11c24948aa51623c6ca8d)